test/core/componentTests.ts

```diff
@@ -244,7 +244,7 @@ describe("Component behavior", () => {
     svg.remove();
   });
 
-  it("hitboxes are created iff there are registered interactions", () => {
+  it("hitboxes are created iff there are registered interactions (that require hitboxes)", () => {
```

Comment:
The way I'm reading 
> "hitboxes are created iff there are registered interactions (that require hitboxes)"
is hitboxes are required for registered interactions and that the hitbox requirement is sort of an afterthought

whereas
> "hitboxes are created iff there are registered interactions that require hitboxes"

to me reads as only those interactions that require will have the creation of hitboxes.